pages/components/search-panel/index.jsx

```diff
@@ -39,7 +40,10 @@ class SearchPanel extends Component {
 }
 
 SearchPanel.defaultProps = {
-  searchableModels: ['establishments', 'profiles', 'projects']
+  searchableModels: ['establishments', 'profiles', 'projects'],
+
+  // action should only be provided if the search is being submitted to a page other than the current page
+  action: undefined
```

Comment:
There's no need to default something to undefined. If things are undefined then they're already `undefined`.